Resources/Locale/en-US/_RMC14/chat/highlights.ftl

```diff
@@ -13,7 +13,7 @@ highlights-staff-officer = Staff Officer, "SO", Combat Information Center, "CIC"
 
 # Requisitions
 highlights-quartermaster = Quartermaster, "QM", Requisitions, "Req", Command, Supply, "Drop", Crate, Budget, Ammo
-highlights-cargo-technician = Cargo Technician, Cargo Tech, "CT" Requisitions, "Req", Supply, "Drop", Crate, Budget, Ammo
+highlights-cargo-technician = Requisitions Technician, Req Technician, Requisitions Tech, Req Tech, "RT", Requisitions, "Req", Supply, "Drop", Crate, Budget, Ammo
```

Comment:
This should probably keep the CT highlights too at least for a while until people get used to the change
```suggestion
highlights-cargo-technician = Requisitions Technician, Req Technician, Requisitions Tech, Req Tech, "RT", Requisitions, "Req", Supply, "Drop", Crate, Budget, Ammo, Cargo Technician, Cargo Tech, "CT"
```